src/core/adaptors/verifications/index.adaptor.ts

```diff
@@ -179,3 +182,18 @@ export const getVerificationHistory = async (
     return { data: null, error: 'Error in getVerificationIndividuals API call' };
   }
 };
+
+export const createVerificationIndividualAdaptor = async (
+  verificationId: string,
+): Promise<AdaptorRes<VerificationIndividualRes>> => {
+  try {
+    const payload: VerificationIndividualReq = {
+      customer_id: uuidv4(),
```

Comment:
why we generate unique id for each customer?
don't we have from BE?
is it fixed for each verfication_id?

Reply:
It is a solution from BE team for two-layer verification, since in BE, each link is created for a single verification individual, we assume that this individual with this Unique GUID should be created in BE first, and then BE returns the link

Reply:
I asked Ehsan about the customer id
It doesn't need to use unique id. It just should be a random with 10 characters 
(In BE it's static too not unique)